src/views/Timesheets/index.js

```diff
@@ -62,7 +62,48 @@ const Timesheets = (props) => {
   const [saving, setSaving] = useState(false);
   const [snackbarText, setSnackbarText] = useState('');
   const [snackbarSeverity, setSnackbarSeverity] = useState('');
+  const [clockInOut, setClockInOut] = useState('Clock In');
+  
+  // disabled lint in some lines in order to remove warning about race condition that does not apply
+  // in our current case.
+  // eslint-disable-next-line
+  useEffect(async ()=>{
+    // updates ui with the current status of the users clocked in feature
+    // either clocked in and ready to clock out or the apposite.
+    async function getClockInOutStatus(){
+      try{
+            let status = await jobs.clockOut();
+            
+            if(status[0].currentState){
+              setClockInOut("Clock Out");
+              //
```

Comment:
I return data using the <IEnumerable> type which is a list of lists. However I only really have 1 list in that list. The benefit from using it come in the back end when we are parsing data that comes back from the database. it lets us specify exactly what we a want to do with every piece of data coming from the data. The downside to it is that I have to reference what I get back in the front end with a [0] so that it knows i'm talking about the first item in the list. Even though there's only 1.

Reply:
added a comment about status and what it holds